packages/fhir-converter/src/templates/cda/Resources/Encounter.hbs

```diff
@@ -46,9 +46,19 @@
                 {{{toJsonString this}}},
             {{/each}}
         ],
-        
-        "class":{{>DataType/Coding.hbs code=encounter.code}},
-        
+
+        {{#if encounter.code.code}}
+            "class":{{>DataType/Coding.hbs code=encounter.code}},
+        {{else}}
+          {{#if encounter.code.translation}}
+            "class":{{>DataType/Coding.hbs code=encounter.code.translation }},
+          {{else}}
+            "class": {
+              "display": "n/a"
+            }
```

Comment:
why dont you just update coding.ts to display the unk value if there was no defined code in the code passed to it and then we can leave the template as it was before? here and elsewhere

Reply:
I thought about it but we only want it adding unknown values for fields that are mandatory. If we add this logic to `CodeableConcept` we will render a lot more unknowns than we need too

Reply:
You could add a new CodeableConcept primitive for when you want it to use unknowns if not defined. Or pass in a boolean flag etc